package/aarch64/Ubuntu_16.04/package.sh

```diff
@@ -1,28 +1,14 @@
 #!/bin/bash -e
 
 readonly VE_LOCATION=/tmp/reqExec_pkg_ve
-readonly PYINSTALLER_DIR=/tmp/pyinstaller
-readonly PYINSTALLER_AARCH_FIX_REPO="https://github.com/Bharath92/pyinstaller.git"
```

Comment:
pyinstaller 3.3.1 has my changes to build aarch64 binary. so we can install pyinstaller 3.3.1 directly